packages/dashboard/src/API.tsx

```diff
@@ -7,6 +7,7 @@ import type { Services as ServiceType } from '../../mask/dist/src/extension/serv
 import type { MaskEvents } from '@masknet/shared-base'
 import type { WebExtensionMessage } from '@dimensiondev/holoflows-kit'
 import type { WalletMessages } from '@masknet/plugin-wallet'
+import type { PluginTransakMessages } from '@masknet/plugin-transak'
```

Comment:
```suggestion
import type { TransakMessages } from '@masknet/plugin-transak'
```

Reply:
I think we should not add the message manager of an individual plugin to PluginMessages anymore. Let's import it from its package and use it directly.

```diff
diff --git a/packages/dashboard/src/components/PageFrame/FeaturePromotions/index.tsx b/packages/dashboard/src/components/PageFrame/FeaturePromotions/index.tsx
index b9db9eee3..ae45d0afd 100644
--- a/packages/dashboard/src/components/PageFrame/FeaturePromotions/index.tsx
+++ b/packages/dashboard/src/components/PageFrame/FeaturePromotions/index.tsx
@@ -3,7 +3,8 @@ import { useNavigate } from 'react-router-dom'
 import { makeStyles } from '@masknet/theme'
 import { useRemoteControlledDialog } from '@masknet/shared'
 import { useAccount } from '@masknet/web3-shared-evm'
-import { PluginMessages, Services } from '../../../API'
+import type { TransakMessages } from '@masknet/plugin-transak'
+import { Services } from '../../../API'
 import { PersonaContext } from '../../../pages/Personas/hooks/usePersonaContext'
 import { DashboardRoutes } from '@masknet/shared-base'
 import { PluginId } from '@masknet/plugin-infra'
@@ -37,7 +38,7 @@ export const FeaturePromotions = memo(() => {
     const account = useAccount()

     const { currentPersona, connectPersona } = PersonaContext.useContainer()
-    const { setDialog: setBuyDialog } = useRemoteControlledDialog(PluginMessages.Transak.buyTokenDialogUpdated)
+    const { setDialog: setBuyDialog } = useRemoteControlledDialog(TransakMessages.buyTokenDialogUpdated)

     const isConnectedTwitter = useMemo(() => {
         if (!currentPersona) return false

```

Reply:
the dashboard is intended to be an isolated project (although the intention is failing, we should not make it worse). it should not directly access other packages in this way.